draft-nygren-httpbis-httpssvc.md

```diff
@@ -455,9 +455,31 @@ received over HTTPS to those received via DNS.
 
 # Client behaviors
 
-TODO: Write more details on the recommended resolution algorithm.
-This section does NOT yet cover the SvcRecordType=0 case well yet.
+## Client resolution {#resolution}
 
+When attempting to resolve a name HOST, clients should follow in-order:
+
+1. Issue parallel AAAA, A, and HTTPSSVC queries for the name HOST.
```

Comment:
Technically if you're v4 or v6 only this is obviously not quite right.  I think "A/AAAA" is the usual shorthand.